src/retabled/core.cljc

```diff
@@ -84,11 +84,23 @@
     [:tr.row.screen-controls-row
      [:td.cell.screen-controls {:colSpan num-columns}
       left-bar-content
-      [:div.control.first [:a.control-label {:on-click #(set-current-screen 0)} rr-content]]
-      [:div.control.prev [:a.control-label {:on-click #(set-current-screen (prevfn))} r-content]]
+      [:div.control.first [:a.control-label (if (not (= (get-current-screen) 0))
```

Comment:
Code style: don't use (if (not)). Use if-not https://clojuredocs.org/clojure.core/if-not